plugins/node/opentelemetry-plugin-ioredis/src/utils.ts

```diff
@@ -60,21 +65,35 @@ export const traceConnection = (tracer: Tracer, original: Function) => {
   };
 };
 
-export const traceSendCommand = (tracer: Tracer, original: Function) => {
+const defaultDbStatementSerializer: DbStatementSerializer = (
+  cmdName,
+  cmdArgs
+) =>
+  Array.isArray(cmdArgs) && cmdArgs.length
+    ? `${cmdName} ${cmdArgs.join(' ')}`
+    : cmdName;
+
+export const traceSendCommand = (
+  tracer: Tracer,
+  original: Function,
+  config?: IORedisPluginConfig
+) => {
   return function(
     this: ioredisTypes.Redis & IORedisPluginClientTypes,
     cmd?: IORedisCommand
   ) {
     if (arguments.length >= 1 && typeof cmd === 'object') {
+      const dbStatementSerializer =
```

Comment:
I think this assignment for `dbStatementSerializer` should be done one level up (between line 80 and 81) so that it is assigned once only during the `patch` function

Reply:
Ah yes I like this optimization

Reply:
Done